django/chat/models.py

```diff
@@ -411,7 +411,7 @@ class ChatFile(models.Model):
     def __str__(self):
         return f"File {self.id}: {self.filename}"
 
-    def extract_text(self, fast=True):
+    def extract_text(self, pdf_method="default"):
```

Comment:
Instead of just "fast" (local with pdfium) and "not fast" (azure prebuilt-layout), we can supply a string for pdf_method.

Now, these will be one of:
* default (pdfium)
* azure_read
* azure_layout

We can easily add or change methods in the future.